library/src/G4CMPPhononElectrode.cc

```diff
@@ -60,7 +61,7 @@
 // Constructor and destructor
 
 G4CMPPhononElectrode::G4CMPPhononElectrode()
-  : G4CMPVElectrodePattern(), kaplanQP(0) {;}
+  : G4CMPVElectrodePattern(), G4CMPBoundaryUtils(nullptr), kaplanQP(0) {;}
```

Comment:
This is where I'd suggest having a separate constructor, and passing the name of the electrode directly, instead of passing a null pointer.